FHICORC/ViewModels/QrScannerViewModels/QRScannerViewModel.cs

```diff
@@ -241,6 +241,17 @@ private async Task OnScanSuccess(TokenValidateResultModel tokenValidateResultMod
                 anyTestResults = cwtPayload.DCCPayloadData.DCC.Tests?.Any() ?? false;
                 anyRecovery = cwtPayload.DCCPayloadData.DCC.Recovery?.Any() ?? false;
             }
+            else if (tokenValidateResultModel.DecodedModel is Core.Services.Model.SmartHealthCardModel.Shc.SmartHealthCardWrapper shc)
+            {
+                if (shc.SmartHealthCard.VerifiableCredential.CredentialSubject.Immunizations?.Any() ?? false)
```

Comment:
What happens if we dont have any immunizations? 
Will it show an error screen, or will it do nothing? If nothing, should it show an error screen instead, so user gets feedback?

Reply:
Discussed, will show invalid screen :)